astropy/io/ascii/misc.py

```diff
@@ -0,0 +1,128 @@
+"""A Collection of useful miscellaneous functions.
```

Comment:
They would need their own unit tests and numpy-compliant docstrings, etc.  For now just leave them here.